spec/lib/danger/ci_sources/jenkins_spec.rb

```diff
@@ -86,7 +86,7 @@
         expect(described_class.validates_as_ci?(valid_env)).to be true
       end
 
-      it "doesnt validate when require env variables are not set" do
+      it "doesn't validate when require env variables are not set" do
```

Comment:
No, nothing further than anything inside `rubocop.yml` - personally, I have no problem with anyone's choices, so long as they're readable and easily understood. 

I'd be hesitant to add anything that cannot be automated, given that I write most tests in the app - and I'm the least likely to conform :P